src/doc/rustc/src/platform-support/hexagon-unknown-qurt.md

```diff
@@ -0,0 +1,165 @@
+suitable for both static images and dynamic shared objects.
+
+## Building the target
+
+You can build Rust with support for the target by adding it to the `target`
+list in `bootstrap.toml`:
+
+```toml
+[build]
+build-stage = 1
+host = ["<target for your host>"]
+target = ["<target for your host>", "hexagon-unknown-qurt"]
+
+[target.hexagon-unknown-qurt]
+cc = "hexagon-clang"
+cxx = "hexagon-clang++"
+ranlib = "hexagon-ranlib"
+ar = "hexagon-ar"
+llvm-libunwind = 'in-tree'
+```
+
+Replace `<target for your host>` with `x86_64-unknown-linux-gnu` or whatever
+else is appropriate for your host machine.
+
+## Building Rust programs
+
+Rust does not yet ship pre-compiled artifacts for this target. To compile for
+this target, you will either need to build Rust with the target enabled (see
+"Building the target" above), or build your own copy of `core` by using
+`build-std` or similar.
+
+## Static Image Targeting
+
+For static executables that run directly on QuRT, use the default target
+configuration with additional linker flags:
+
+```sh
+# Build a static executable for QuRT
+cargo build --target hexagon-unknown-qurt \
+    -C link-args="-static -nostdlib" \
+    -C link-args="-L/opt/Hexagon_SDK/6.3.0.0/rtos/qurt/computev69/lib" \
+    -C link-args="-lqurt -lc"
+```
+
+This approach is suitable for:
+- Standalone QuRT applications
+- System-level services
+- Boot-time initialization code
+- Applications that need deterministic memory layout
+
+## User-Loadable Shared Object Targeting
+
+For shared libraries that can be dynamically loaded by QuRT applications:
+
+```sh
+# Build a shared object for QuRT
+cargo build --target hexagon-unknown-qurt \
+    --crate-type=cdylib \
+    -C link-args="-shared -fPIC" \
+    -C link-args="-L/opt/Hexagon_SDK/6.3.0.0/rtos/qurt/computev69/lib"
+```
+
+This approach is suitable for:
+- Plugin architectures
+- Runtime-loadable modules
+- Libraries shared between multiple applications
+- Code that needs to be updated without system restart
+
+## Configuration Options
+
+The target can be customized for different use cases:
+
+### For Static Images
+```toml
+# In .cargo/config.toml
+[target.hexagon-unknown-qurt]
+rustflags = [
+    "-C", "link-args=-static",
+    "-C", "link-args=-nostdlib",
+    "-C", "target-feature=-small-data"
+]
+```
+
+### For Shared Objects
+```toml
+# In .cargo/config.toml
+[target.hexagon-unknown-qurt]
+rustflags = [
+    "-C", "link-args=-shared",
+    "-C", "link-args=-fPIC",
+    "-C", "relocation-model=pic"
+]
+```
+
+## Testing
+
+Since `hexagon-unknown-qurt` requires the QuRT runtime environment, testing requires
+either:
+- Hexagon hardware with QuRT
+- `hexagon-sim`
+- QEMU (`qemu-system-hexagon`)
+
+## Cross-compilation toolchains and C code
+
+This target requires the [Hexagon SDK toolchain for C interoperability](https://softwarecenter.qualcomm.com/api/download/software/sdks/Hexagon_SDK/Linux/Debian/6.3.0.0/Hexagon_SDK.zip):
```

Comment:
What is the license of this SDK? Maybe you could link to the page that links to the zip file and presumably also the license.